homework-submission/week5/practice-app/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import Login from './components/login';
+import HomeScreen from './components/HomeScreen';
+import AddPhotos from './components/AddPhotos';
+import GetInfo from './services/GetInfo';
+import './App.css';
+
+class App extends Component {
+  state = {
+    userName: '',
+    isLoggedIn: false,
+    mainInfo: [],
+  };
+
+  handelLogin = userLoginInfo => {
+    this.setState({
+      userName: userLoginInfo,
+      isLoggedIn: true,
+    });
+  };
+
+  componentDidMount() {
+    GetInfo.getMainInfo().then(mainInfo =>
+      this.setState({
+        mainInfo,
+      }),
```

Comment:
here you are fetching the data after loading the page, means:
even if the user doesn't logged in
so why do i need to import the data from the server before log in